selfdrive/controls/lib/events.py

```diff
@@ -576,9 +563,34 @@ def joystick_alert(CP: car.CarParams, CS: car.CarState, sm: messaging.SubMaster,
     ET.PERMANENT: NormalPermanentAlert("GPS Malfunction", "Likely Hardware Issue"),
   },
 
-  EventName.localizerMalfunction: {
-    ET.NO_ENTRY: NoEntryAlert("Localizer Malfunction"),
-    ET.SOFT_DISABLE: soft_disable_alert("Localizer Malfunction"),
+  EventName.locationdTemporaryError: {
```

Comment:
I'd just rename the old events to the new temporary ones instead of deprecating since they're effectively the same and doesn't break old logs

Reply:
Do you mean in the serial struct?

Reply:
yes